src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -111,6 +112,59 @@ public function testGuessMaxLengthForConstraintWithMinValue()
         $this->assertNull($result);
     }
 
+    /**
+     * Check if FileType contains accept attribute.
+     */
+    public function testGuessMimeTypesForConstraintWithMimeTypesValue()
+    {
+        $mineTypes = ['image/png', 'image/jpeg'];
+        $constraint = new File(['mimeTypes' => $mineTypes]);
+        $typeGuess = $this->guesser->guessTypeForConstraint($constraint);
+        $this->assertInstanceOf('Symfony\Component\Form\Guess\TypeGuess', $typeGuess);
+        $this->assertArrayHasKey('attr', $typeGuess->getOptions());
+        $this->assertArrayHasKey('accept', $typeGuess->getOptions()['attr']);
+        $this->assertEquals(implode(',', $mineTypes), $typeGuess->getOptions()['attr']['accept']);
+    }
+
+    /**
+     * Check if file assert not contains mime types.
+     */
+    public function testGuessMimeTypesForConstraintWithoutMimeTypesValue()
+    {
+        $constraint = new File();
+        $typeGuess = $this->guesser->guessTypeForConstraint($constraint);
+        $this->assertInstanceOf('Symfony\Component\Form\Guess\TypeGuess', $typeGuess);
+        if (isset($typeGuess->getOptions()['attr'])) {
+            $this->assertArrayNotHasKey('accept', $typeGuess->getOptions()['attr']);
```

Comment:
Is this line ever executed? shouldn't we check by empty options instead?

Reply:
I modified, but I was left on the idea that it could have a validator element that can be useful in front, but from what I see in the doc, there is nothing except "mime types" which becomes "accept".